pkg/util/grpc/util.go

```diff
@@ -0,0 +1,17 @@
+package grpc
+
+import (
+	"github.com/gogo/status"
+	"google.golang.org/grpc/codes"
+)
+
+// IsGRPCContextCanceled returns whether the input error is a GRPC error wrapping
+// the context.Canceled error.
+func IsGRPCContextCanceled(err error) bool {
```

Comment:
Is this used anywhere?

Reply:
So far it's used here:
https://github.com/cortexproject/cortex/pull/2242/files#diff-5c63c6fec32b3d7499095ed68c1ea1caR150

Reply:
Thanks. I clicked “viewed” checkbox and then couldn’t find it. 🤦‍♂